azure-keyvault-cryptography/src/main/java/com/microsoft/azure/keyvault/cryptography/ByteExtensions.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.keyvault.cryptography;
+
+public final class ByteExtensions {
+
+    public static boolean sequenceEqualConstantTime( byte[] self, byte[] other )
```

Comment:
For now, I moved the method to the class that requires it and removed it from the list of publicly accessible ByteExtensions methods until we find other use cases for it.